### PR TITLE
[TM-4845] Position Classifications Integration

### PR DESCRIPTION
**Pull request by jlam-mpc**
Dual Merge: 
- [FE PR](https://github.com/MetaPhase-Consulting/State-TalentMAP/pull/2680)
- [Mock PR](https://github.com/MetaPhase-Consulting/mock-fsbid/pull/405)

[TM-4845](https://metaphase.atlassian.net/browse/TM-4845)

[TM-4845]: https://metaphase.atlassian.net/browse/TM-4845?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: talentmap_api/fsbid/services/position_classifications.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        "I_ORG_CODE": '',
+        "I_POS_NUM_TXT": request,
+        "I_POS_OVRSES_IND": '',
+        "I_PS_CD": '',
+        "I_BUREAU_CD": '',
+        "I_PUBS_CD": '',
+        "I_JC_ID": '',
+        "I_ORDER_BY": '',
+        "I_PCT_CODE": ''
+    }
```
Comment (scott062) [inline talentmap_api/fsbid/services/position_classifications.py:38]: Error said it needed two filters but just one worked 🤷

File: talentmap_api/fsbid/urls/position_classifications.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+from django.conf.urls import url
+from rest_framework import routers
+
+from talentmap_api.fsbid.views import position_classifications as views
+
+router = routers.SimpleRouter()
+
+urlpatterns = [
+    url(r'^(?P<pk>\w{2,3}[0-9]+)/$', views.FSBidPositionClassificationsView.as_view(), name='FSBid-position-classifications'),
```
Comment (pveland621) [inline talentmap_api/fsbid/urls/position_classifications.py:9]: curious - what does the `\w{2,3}` do in this regex? i know the usual pattern we do for numbers in the url is just `?P<pk>[0-9]+`

**Review by pveland621 — COMMENTED** [review]
this integration setup looks good to me

File: talentmap_api/fsbid/urls/position_classifications.py (hunk 2 above)
Comment (scott062) [reply talentmap_api/fsbid/urls/position_classifications.py:9]: image
Check out this site for regex stuff - @elizabeth-jimenez showed it to me

**Review by M00staff — APPROVED** [review]
looks good to me, as well

**Pull request merged**